dockerfiles/entrypoints/proxito.sh

```diff
@@ -2,13 +2,12 @@
 
 ../../docker/common.sh
 
-if [ -n "${DOCKER_NO_RELOAD}" ];
-then
-    RELOAD='--noreload'
-    echo "Running Docker with no reload"
+CMD="python3 manage.py runserver 0.0.0.0:8000"
```

Comment:
If you want to use `nodemon` for reloading instead, you always have to pass `--noreload` to Django command. Otherwise, there is going to be 2 process watching this for reloading.

Reply:
Oh hrm, I missed this, I thought the option was `--reload` :thinking:

Now I'm wonder why this fix solved anything

Reply:
Updated here at least, however I think the ultimate fix is setting higher ulimits in the docker compose config, which oddly didn't seem to have any effect. I did just have luck setting high `nofile` at the system level and restarting docker though.